packages/build-utils/src/convert-runtime-to-plugin.ts

```diff
@@ -50,6 +60,15 @@ export function convertRuntimeToPlugin(
       // @ts-ignore This symbol is a private API
       const lambdaFiles: Files = output[FILES_SYMBOL];
 
+      // `.output` was already created by the Build Command, so we have
+      // to ensure its contents don't get bundled into the Lambda. Similarily,
+      // we don't want to bundle anything from `.vercel` either.
+      for (const file in lambdaFiles) {
+        if (file.startsWith('.output') || file.startsWith('.vercel')) {
+          delete lambdaFiles[file];
+        }
+      }
```

Comment:
I guess in this case we can ignore them in the initial glob already, no? If we iterate over all files again it could slow down quite a bit, if we do it at the time we glob, we don't have to iterate again.

Reply:
Great point!

I thought the glob would be inside the Runtime, but it's actually outside, so I'll do that.